src/main/resources/docker/Dockerfile.template

```diff
@@ -0,0 +1,21 @@
+# based on https://github.com/rastasheep/ubuntu-sshd/blob/master/18.04/Dockerfile
```

Comment:
Does it mean we are kinda forking this Dockerfile? Why do we want to keep copy of it here? Is it suggested way written somewhere in the `rastasheep` repository?

Reply:
It's simple enough not to include the dependency. Like including whole http library or just implementing one function